53.md

```diff
@@ -77,7 +77,7 @@ Event `kind:1311` is live chat's channel message. Clients MUST include the `a` t
 
 ## Use Cases
 
-Common use cases include meeting rooms/workshops, watch-together activities, or event spaces, such as [live.snort.social](https://live.snort.social) and [nostrnests.com](https://nostrnests.com).
+Common use cases include meeting rooms/workshops, watch-together activities, or event spaces, such as [zap.stream](https://zap.stream).
```

Comment:
Why nostrnests.com was removed? Is it dead?

Reply:
Uses a different event kind. They will probably write a NIP or modify the live events one to add it. Or I will in a different PR.